core/templating/templating.go

```diff
@@ -47,6 +50,7 @@ func NewTemplator() *Templator {
 		raymond.RegisterHelper("randomIPv4", t.randomIPv4)
 		raymond.RegisterHelper("randomIPv6", t.randomIPv6)
 		raymond.RegisterHelper("randomUuid", t.randomUuid)
+		raymond.RegisterHelper("Request.body jsonPath", t.jsonPath)
```

Comment:
Is it supposed to register {{ Request.Body xpath }} as well?

Reply:
This was an old attempt, its not used or required, should be deleted.